src/navigation-instruction.js

```diff
@@ -245,7 +245,7 @@ export class NavigationInstruction {
     }
   }
 
-  _buildTitle(separator: string = ' | '): string {
+  _buildTitle(): string {
```

Comment:
removing the argument is a potential breaking change

Reply:
fair enough, I will restore that one :)